spec/lib/heroku-scaler/web_scale_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+require 'heroku-scaler/web_scale'
+
+module HerokuScaler
+  class TestConfig
+    def app_name
+      "bonanza"
+    end
+
+    def big_dyno_type
+      "big-boy"
+    end
+
+    def big_dyno_count
+      99
+    end
+
+    def small_dyno_type
+      "small-boy"
+    end
+
+    def small_dyno_count
+      9
+    end
+
+    def auth_token
+      "some-token"
+    end
+  end
+
+  class TestPlatformAPI
+    def self.connect_oauth(token)
+      TestHeroku.new
+    end
+  end
+
+  class TestHeroku
+    def formation
+      TestFormation.new
+    end
+  end
+
+  class TestFormation
+    def update(app, type, hash)
+    end
+  end
+
+  describe WebScale do
```

Comment:
Yeah.  Will do at some point soon.  Per the README, the weekday? business won't work in some time zones.  I'd like to fix this soon.